BlinkID/example/lib/document_scanner.dart

```diff
@@ -0,0 +1,173 @@
+import 'package:blinkid_flutter/microblink_scanner.dart';
+import 'package:blinkid_flutter/microblink_scanner_widget.dart';
+import 'package:flutter/material.dart';
+import 'package:permission_handler/permission_handler.dart';
+
+typedef MicroblinkScannerResultCallback = Future<bool> Function(List<RecognizerResult>?);
+
+typedef MicroblinkRecognizerResult = List<RecognizerResult>;
```

Comment:
Name is quite long. I don't think "Mircoblink" brings anything useful. Could be simply `ScanResult`.